src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,48 @@
+/* Used Conversation.java a reference */
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+public class Activity {
+
+	/* String type
+ 	 * newUser - user has joined
+ 	 * newConvo - conversation has been made
+ 	 * newMessage - message has been sent
+ 	*/
+	public final String type;
+	public final UUID id;
+	public final Instant creation;
+	
+	/*
+	*	Constructs a new activity
+	*	
+	* @param type is the type of activity made
+	*	@param id is the ID of who ever made the activity
+	*	@param creation is the creation time of this activity
+	*/
+	public Activity(String type, UUID id, Instant creation) {
+		this.type = type;
+		this.id = id;
+		this.creation = creation;
```

Comment:
Is the type supposed to be something like "message" or "user created" or something like that? What is type typically going to be?

Reply:
Wondering the same thing ^ Is the type being passed in so that you can narrow down the activity to message activity, conversation activity, etc?

Reply:
Yepp, type is a string that's supposed to help differentiate what type of activity it is. So there's three types: newUser, newConvo, newMessage. I believe I added a comment in the same file that says this.